resources/assets/js/components/HRPortal/JobIndexHr.tsx

```diff
@@ -94,13 +94,13 @@ const CompletedJobsAccordion: React.FC<CompletedJobsAccordionProps> = ({
   );
 };
 
-interface JobIndexPageProps {
+interface JobIndexHrProps {
```

Comment:
Is there going to be a functional difference between the HR Portal index and any other index? I'd also like to come up with some sort of naming convention to follow for these types of files.

On the JPB we have the parent folder containing the sections, and if each section has a need for multiple files we subfolder those... the "container" component have the `SectionPage` naming convention in those sections, so I'd like to either follow that or figure out something else.

Reply:
The Job index on all three portals is distinct. Only HR can claim jobs, Managers can edit them, and applicants can only view the final poster.

I've been following the `___Page` for containers patterns. I don't see JobIndexHr as the container component, I've created a JobIndexHrPage component in the other branch I'm working on.